ecommerce/frontend/package.json

```diff
@@ -9,6 +9,7 @@
         "@testing-library/user-event": "^12.2.0",
         "antd": "^4.8.0",
         "axios": "^0.21.0",
+        "classnames": "^2.2.6",
```

Comment:
I have merged a `package.json` that contains classnames before, why is it showing it as new?

Reply:
I might have branched out of master before you